db/schema.rb

```diff
@@ -82,6 +82,12 @@
     t.index ["vendor_id"], name: "index_products_on_vendor_id"
   end
 
+  create_table "tags", force: :cascade do |t|
```

Comment:
Why are tags and vendor_tags tables in the database? I think they should have been removed in the last pull request.